docs/datascience/jupyter-kernel-options.md

```diff
@@ -0,0 +1,45 @@
+---
+Order: 7
+Area: datascience
+TOCTitle: Jupyter Kernel Options in VS Code
+ContentId: 3b6da7e6-c449-4c62-a019-9202412aac04
+PageTitle: Jupyter Kernel Options in VS Code
+DateApproved: 11/21/2022
+MetaDescription: Descriptions, tips, and tricks on kernel selection options when working with Jupyter Notebooks in Visual Studio Code.
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Jupyter Kernel Options in VS Code
+
+The VS Code notebooks’ kernel picker lists the following categories of connection methods to run the code in your Jupyter notebook:
+- Local Jupyter Kernels
```

Comment:
We now call them "Jupyter Kernels" in the extension.

Reply:
Fixed